videotroncmdb.json

```diff
@@ -0,0 +1,302 @@
+{
+    "appid": "e9279bd9-6fbf-4aae-9982-83109d9e5423",
+    "name": "Videotron CMDB",
+    "description": "This app integrates with the Videotron CMDB tool to perform lookups",
+    "type": "information",
+    "product_vendor": "Videotron",
+    "logo": "logo_videotroncmdb.svg",
+    "logo_dark": "logo_videotroncmdb_dark.svg",
+    "product_name": "CMDB",
+    "python_version": "3",
+    "product_version_regex": ".*",
+    "publisher": "Splunk",
```

Comment:
This is a community app right? it should be `Splunk Community`

Reply:
Yes, Updated publisher as `Splunk Community`.